front-end-poc-pmml-linear-model-viewer/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@manstis/pmml-linear-model-viewer",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "cxsd": "cxsd",
+    "clean": "gulp clean:output",
+    "clean:build": "yarn run bootstrap && tsc",
+    "generate-cxsd": "cxsd --out-js ./src/generated --out-ts ./src/generated http://dmg.org/pmml/v4-4/pmml-4-4.xsd",
+    "bootstrap": "yarn run clean && yarn run generate-cxsd",
+    "build": "tsc"
```

Comment:
I would add tsc as devDependency so you don't need to install it globally to build